indigo_app/views/tasks.py

```diff
@@ -132,8 +132,17 @@ def get_context_data(self, **kwargs):
         Task.decorate_potential_assignees([task], self.country)
         Task.decorate_permissions([task], self)
 
+        # add work to context
+        if task.work:
+            context['work'] = task.work
+
         return context
 
+    def get_template_names(self):
+        names = super().get_template_names()
```

Comment:
1. rather than calling super() and then ignoring it, do the if statement first, then call super() if necessary.
2. for symmetry, please rename `work_task.html` to `work_task_detail.html`.